/home/pmos/build/src/android_kernel_huawei_kiwi-6e870147270171fb00496ac229ffa5eb86a496d1/drivers/staging/prima/CORE/VOSS/src/wlan_nv_template_builtin.c:590:10: error: initializer element is not computable at load time
  590 |          (((char *)&(nvDefaults.tables.hwCalValues.calData.psSlpTimeOvrHdxLNA5G) +
      |          ^
/home/pmos/build/src/android_kernel_huawei_kiwi-6e870147270171fb00496ac229ffa5eb86a496d1/drivers/staging/prima/CORE/VOSS/src/wlan_nv_template_builtin.c:590:10: note: (near initialization for 'NvTablesBuiltIn[4][3].offset')

diff --git a/drivers/staging/prima/CORE/VOSS/src/wlan_nv_template_builtin.c b/drivers/staging/prima/CORE/VOSS/src/wlan_nv_template_builtin.c
index 14c6eadf524..e26b697f793 100644
--- a/drivers/staging/prima/CORE/VOSS/src/wlan_nv_template_builtin.c
+++ b/drivers/staging/prima/CORE/VOSS/src/wlan_nv_template_builtin.c
@@ -587,9 +587,8 @@ _NV_TEMPLATE_TABLE NvTablesBuiltIn[/*TABLES_MAX*/][TABLE_ENTRIES_MAX] = {
          ((char *)&(nvDefaults.tables.hwCalValues.calData.psSlpTimeOvrHdxLNA5G) -
           (char *)&nvDefaults.tables.hwCalValues.calData),"psSlpTimeOvrHdxLNA5G"},
       {"ed",_ID_U8,SINGULAR,0,0,0,
-         (((char *)&(nvDefaults.tables.hwCalValues.calData.psSlpTimeOvrHdxLNA5G) +
-          sizeof(uint16)) - (char *)&nvDefaults.tables.hwCalValues.calData),
-         "nv_TxBBFSel9MHz"},
+         ((char *)&(nvDefaults.tables.hwCalValues.calData.psSlpTimeOvrHdxLNA5G) -
+          (char *)&nvDefaults.tables.hwCalValues.calData),"nv_TxBBFSel9MHz"},
       {"ee",_ID_U8,SINGULAR,0,0,0,
          ((char *)&(nvDefaults.tables.hwCalValues.calData.hwParam2) -
           (char *)&nvDefaults.tables.hwCalValues.calData),"hwParam2"},
